python.spec

```diff
@@ -1,4 +1,4 @@
-### RPM external python 2.7.6
+### RPM external python 2.7.10
```

Comment:
Looks like python 2.7.9 and above has enabled SSL certificate verification by default for stdlib http clients.
This breaks at least the das_client.py 
```urllib2.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)>```
I am sure it will break many of the computing workflows :-(